App.js

```diff
@@ -204,26 +204,7 @@ function ChatScreen({route, navigation}) {
           <Text style={{color: 'blue', fontSize: 18, marginBottom: 10}}>Link to another chat</Text>
         </Pressable>
         <Pressable
-          onPress={() => {
-            // This is fairly complex but seems to be the way we can "push" a new stack on and add the Settings screen in the routes
-            navigation.dispatch((state) => {
-              const newState = {...state};
-              const routes = [...newState.routes];
-              routes.push({
-                name: 'SettingsStack',
-                state: {
-                  index: 1,
-                  routes: [{name: 'Settings'}, {name: 'About'}]
-                }
-              });
-              const newestState = {
-                ...newState,
-                routes,
-                index: routes.length - 1,
-              };
-              return CommonActions.reset(newestState);
-            });
-          }}
+          onPress={() => navigation.push('SettingsStack', { screen: 'About' })}
```

Comment:
You can update this to `navigation.push('SettingsStack', {screen: 'About', initial: false});` that way when you do `up` it will not go back to the chat screen rather it will go to settings which is what is expected.
You can read about that additional property here: https://reactnavigation.org/docs/nesting-navigators#rendering-initial-route-defined-in-the-navigator

Reply:
I am saw that but even now without the initial: false the navigation goes to Settings 🤔 does it not work for you?

Reply:
With `navigation.replace` in the About component then it will create an issue. You can test it by navigating from `LHNHeader avatar` -> Settings -> About and then clicking up. When you will be on the Settings screen you will need to click `up` twice. 
If you update the code in the About component to use `pop` then it wouldn't work with the `up` click in the About page because it would remove the whole nested route. So to fix this I tried with `initial: false` and that seem to solve both the issue.

Reply:
Oh, cool, thanks for the reproduction steps, I will add this

Reply:
You don't need `replace` anymore in About because `initial: false` will handle adding `Settings` in routes. Can you try that?

Reply:
ah thanks!